app/src/app/api/v0/sector/[sectorId]/required-scopes/route.ts

```diff
@@ -0,0 +1,29 @@
+import { db } from "@/models";
+import { apiHandler } from "@/util/api";
+import createHttpError from "http-errors";
+import { NextRequest, NextResponse } from "next/server";
+import { QueryTypes } from "sequelize";
+
+const getRequiredScopes = async (sectorId: string) => {
```

Comment:
This method could be moved to GPCService (as a static class method), as that's part of that domain.